testutil/ciao-down/host_linux.go

```diff
@@ -32,11 +32,15 @@ func getMemAndCpus() (mem int, cpus int) {
 	cpus = getOnlineCPUs() / 2
 	if cpus < 0 {
 		cpus = 1
+	} else if cpus > 8 {
+		cpus = 8
 	}
```

Comment:
Github reviews don't let you comment on the commit message but you might want to s/MB/GB/ in the commit message.